RIS/astm-uam-psu-ris-0.0.0.yaml

```diff
@@ -0,0 +1,867 @@
+          $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Altitude'
+        altitude_upper:
+          description: Upper altitude limit of the resource, if any. 
+          $ref: '#../DSS/astm-uam-psu-dss-0.0.0.yaml/components/schemas/Altitude'
+        type:
+          $ref: '#/components/schemas/ResourceType'
+          
+    ResourceStatus:
+      required: 
+        - status
+      properties:
+        status: 
+          type: string
+          description: The current status of the resource. 
+          enum: 
+          - ACTIVE
+          - INACTIVE
+    ResourceType:
+      required: 
+        - resource_type
+      properties:
+        resource_type: 
+          type: string
+          description: The type of resource.  
+          enum: 
+          - WAYPOINT
+          - VOLUME
+          # TODO: Do we want to keep this generic or more specific and include intersection, merge, etc. ?
+    CapacityLimitType:
+      required:
+        - capacity_type
+      type: object
+      properties:
+        capacity_type:
+          type: string
+          description: |
+            The type of capacity limitation which is to be met for the resource. 
+          externalDocs:
+            url: TBD
+          enum: 
+          - FLOW_RATE_CONSTANT
+          - FLOW_RATE_FIXED_BIN
+          - INSTANTANEOUS
+    CapacityPropertyType:
+      required:
+        - capacity_property
+        - capacity_value
+      type: object
+      properties:
+        capacity_property:
+          type: string
+          description: |
+            Property of the capacity check which must be met.
+            - RATE describes the number of crossings (or events) allowed per time interval DURATION (seconds).
+              Resources using RATE property must also specify a DURATION property. 
+            - DURATION describes the length of a capacity bin or the interval duration (seconds). 
+            - LIMIT describes the max aircraft allowed in either a time bin or instantaneously. If
+              describing a time bin, the LIMIT and DURATION properties must both be provided.  
+          externalDocs:
+            url: TBD
+          enum: 
+          - DURATION
+          - LIMIT
+          - RATE
+        capacity_value:
+          type: number
+          description: |-
+            The number of flights allowed without exceeding the capacity limit.  This
+            number represents the maximum capacity for the limit type. 
+          minimum: 0
+    ResourceCapacity:
+      required:
+      - capacity_limit_type
+      - capacity_property_array
+      properties:
+        capacity_limit_type:
+          $ref: "#/components/schemas/CapacityLimitType"
+        capacity_property_array:
+          minItems: 1
+          maxItems: 2
+          type: array
+          description: |-
+            Describes the properties of the capacity requirements defined for the resource.
```

Comment:
Should this include 
items: 
$ref: "#/components/schemas/CapacityPropertyType"

Reply:
yes, added.